src/pysource/models.py

```diff
@@ -43,6 +43,48 @@ def define(self, dimensions):
         return map_d
 
 
+@switchconfig(log_level='ERROR')
+def initialize_damp_mask(damp, padsizes, spacing, fs=False, abc_type=False):
```

Comment:
Un-necessary duplicate. Should be atwo line chaange. See how it's setup in the devito examples

Reply:
Thanks for the corrections!
All your suggestions were very clear to us.
However, concerning the `initialize_damp`, I could notice that `initialize_damp` function in the Devito and in the JUDI are different. 
Is there any reason for that? I wondering if I could bring the `initialize_damp` from Devito to here. What do you think?

Furthermore, I could see a boundary effect stronger, when I use the `initialize_damp` from the JUDI than when I use it from the Devito. 
See the shot comparison for acoustic case below: 
![JUDIxDEVITO_acoustic_case](https://user-images.githubusercontent.com/30233241/157317126-948efe6c-c676-4584-8519-20bb2ef517fe.png)

Reply:
Well yes the boundaries are a bit different than in Devito because for long simulation (i.e think 20-30sec recording, large 2D model) the devito one becomes a bit unstable. So it damps a little bit less to stay stable and needs a few more point (i.e 60 instead of 40).
I'm sure can make the mask case a bit nicer for your case while still using the same function

Reply:
Got it! I didn't know that the boundary function from Devito (for a long propagation time) is unstable.
We will change the function to support our case.
Thanks for your clarifications!

Reply:
@mloubout 
I could notice that the boundary from JUDI fails in `test_adjoint_J` in Devito for all cases (acoustic, TTI, and viscoacoustic). For example to `dtype=np.float64`, we have:
```
TestAdjoint.test_adjoint_J[layers-shape0-OT2-12-2-acoustic_setup]
INFO     Devito:logger.py:104 <x, J^Ty>: 8551339.045261, <Jx,y>: 8544481.458054, difference: 8.0193e-04, ratio: 1.000803 
```
So considering this fact, could we create a damp kernel so that we could keep the two functions? I suggest this because I would like to have the viscoacoustic operators pass the tests like in Devito.

Reply:
@mloubout please see issue #99